templates/CRM/Contribute/Form/Contribution/Main.tpl

```diff
@@ -91,7 +91,7 @@
   {elseif !empty($ccid)}
     {if $lineItem && $priceSetID && !$is_quick_config}
       <div class="header-dark">
-        {ts}Contribution Information{/ts}
+        {ts}Contribution Information{/ts}  - {ts 1=$display_name}%1{/ts}
```

Comment:
It appears that you have two spaces ahead of the hyphen--there should be just one.  Also, shouldn't this be an en dash (`&ndash;`)?